Makefile

```diff
@@ -161,9 +162,21 @@ $(ZKLOG_OBJDIR)/%.o: src/%.c
 	@mkdir -p $(@D)
 	gcc -o $@ -c $(ZKLOG_CFLAGS) $<
 
+.PHONY: deps
+deps $(TAP_EXEC): | $(REPO_DEPS) $(NPM_EXEC)
+	$(NPM_ENV) $(NPM) install
+
+.PHONY: ensure-node-v6-or-greater-for-test-suite
+ensure-node-v6-or-greater-for-test-suite: | $(TAP_EXEC)
+	@NODE_VER=$(shell node --version) && \
+	    ./node_modules/.bin/semver -r '>=6.x' $$NODE_VER >/dev/null || \
```

Comment:
If you *are* keeping this guard target, then that `semver` binary is only there by luck. One should then add a "semver" dep to "devDependencies".